saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleHandler.java

```diff
@@ -375,11 +391,43 @@ public void handleMQTTCommand(String topic, MqttMessage message) throws MqttExce
                           new byte[] {(byte) 0x00})));
               break;
             default:
-              throw new IOException("Unsupported payload " + message);
+              throw new MqttGatewayException("Unsupported payload " + message);
+          }
+          break;
+        case "refresh/mode":
```

Comment:
Extract constant

Reply:
Will do in the following days. In this case, we should create a constants Class that holds all topics to make it more transparent...